conftest.py

```diff
@@ -96,3 +96,69 @@ def _replace_cassete_uri(vcr_request, uri_map: dict):
     fallback_base_uri = uri_map[vcr_base_uri]
     vcr_request.uri = fallback_base_uri.replace_base_uri(vcr_request.uri)
     return vcr_request
+
+
+def pytest_addoption(parser):
+    """Add custom args to pytest cmdline."""
+    parser.addoption(
+        "--refresh-cassettes",
+        action="store_true",
+        default=False,
+        help="Refresh VCR cassettes.",
+    )
+
+
+def pytest_configure(config):
+    """Customize pytest configuration."""
+    from _pytest.config.exceptions import UsageError
+
+    record_mode_is_none = config.option.record_mode in [None, "none"]
+    if config.option.refresh_cassettes:
+        if record_mode_is_none:
+            config.option.record_mode = "new_episodes"
+        # only execute tests marked with vcr_primer
+        config.option.markexpr = "vcr_primer"
+        num_processes = config.option.numprocesses or 1
+        if num_processes > 1:
+            raise UsageError(
+                "--refresh-cassetes option should not run with multiple processes"
+                " UNLESS the implementation of OCP api tests that depend on"
+                " dynamic client are adapted."
+            )
+    elif not record_mode_is_none:
+        raise UsageError(
+            "Quipucords VCR cassettes should be recorded with --refresh-cassettes"
+            " option explicitly set."
+        )
+    # document quipucords custom markers
+    markers = [
+        "dbcompat: marks tests using our db compat module.",
+        "integration: marks tests as integration tests (deselect with '-m \"not integration\"')",  # noqa: E501
+        "slow: marks tests as slow (deselect with '-m \"not slow\"')",
+        "vcr_primer: tests intended to record vcr cassettes.",
+    ]
+    for mark in markers:
+        config.addinivalue_line("markers", mark)
+
+
+def pytest_collection_modifyitems(config, items):  # pylint: disable=unused-argument
```

Comment:
maybe replace the unused config with `_config`, removing the need for the `pylint: disable=unused-argument directive`.

Reply:
Initially I was afraid pytest needed the function signature intact (in case they passed as kwargs instead of positional arguments) but config is actually in use lol. I'll remove this directive.

Reply:
yeah, that's a pain `should` just be ignored if underscored.  Some folks have written a pylint plugin to get around this: https://github.com/reverbc/pylint-pytest and used by specifying the plugin i.e. `pylint --load-plugins pylint_pytest ...`, I tested with and the warning is no longer given with the underscored parameter.  But this seems overkill for this case so I'm fine with keeping the pylint directive. 👍

Reply:
I think I added the parameter out of necessity because I added this plugin :thinking: 

Opening a new issue to investigate this